lms/static/scripts/frontend_apps/utils/hooks.ts

```diff
@@ -44,8 +44,13 @@ export function usePlaceholderDocumentTitleInDev() {
       return () => {};
     }
 
-    const listener = () => {
-      document.title = 'PLACEHOLDER DOCUMENT TITLE';
+    const listener = (e: NavigateEvent) => {
+      const { pathname: destination } = new URL(e.destination.url);
```

Comment:
```suggestion
      const { pathname: newPath } = new URL(e.destination.url);
```

This makes the `currentPath !== newPath` test more obvious.